.github/workflows/python-package.yml

```diff
@@ -39,4 +39,7 @@ jobs:
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Test with pytest
       run: |
-        python -m pytest tests/
+        python -m pytest tests/test_arg_category.py
```

Comment:
This can be a single

`python -m tests/`

Reply:
There is also the `setup_cli_test.sh` file that should run before the test_cli.py file so that the outputs directory can be ready to store the outputs from argnorm and the current argnorm version can be downloaded. If I am not wrong, pytest shouldn't be able to run the bash file on its own, so I broke `python -m tests/` into different commands.

Reply:
Ah, I now I see the problem. I don't like this set up _at all_: this is very error-prone and a maintenance nightmare.

It really should work with `python -m tests/` and the fact that it doesn't is a sign that you are trying to use `pytest` to do things that really don't work with `pytest`

Reply:
Understood. I've gone ahead and used os.makedirs, and removed the bash script. Now, we can just use `python -m tests/` in the workflow.

Reply:
It's better, but what `test_cli.py` is doing is still not really the kind of thing that we should be using `pytests` for

You can see how it's done in https://github.com/BigDataBiology/SemiBin/ for example: there are tests inside `test` that are pytest-based and tests that call the `SemiBin` script are in `integration-tests`

The big difference is that the integration tests require you to have installed the tool, so the testing workflow is different once you change something in the code

Reply:
I have just separated the tests into tests that use pytest and integration tests. I've also included the integration tests to the workflow. Does this look ok?